scripts/pages/settings/fov.js

```diff
@@ -0,0 +1,66 @@
+function deg2rad(x) {
+	return (x / 180) * Math.PI;
+}
+function rad2deg(x) {
+	return (x * 180) / Math.PI;
+}
+
+class Fov {
+	static panels = {
+		/** @type {SettingsSlider} @static */
+		fov: $('#FOV'),
+		/** @type {TextEntry} @static */
+		horizontalFov: $('#FOV_Horizontal')
+	};
+
+	static loadSettings() {
+		this.updateFOV();
+	}
+
+	static aspectRatio() {
+		// find the display panel
+		// there doesn't seem to be an api for this yet
+		let panel = this.panels.fov;
+		let parent = panel;
+		while ((parent = panel.GetParent())) {
+			panel = parent;
+		}
+
+		return panel.actuallayoutwidth / panel.actuallayoutheight;
+	}
+
+	// based on https://casualhacks.net/Source-FOV-calculator.html
```

Comment:
don't really need this comment

Reply:
I think it should stay. Could be helpful to someone in the future and no harm in keeping it. I didn't have much idea what this system was even doing until I looked at this page.

Reply:
Sorry but unresolving this - why would we link to the source here? It's a useful site that explains the motivation behind what we're doing here.

Reply:
Also wertiop our policy is to let the review resolve conversations

Reply:
Ok, I added it back